CONTRIBUTING.md

```diff
@@ -61,19 +61,13 @@ export MYSQL_CONNECTION_NAME="<PROJECT>:<REGION>:<INSTANCE>"
 export MYSQL_USER="db_user"
 export MYSQL_PASS="db_pass"
 export MYSQL_DB="db_name"
-# MySQL instance with IAM authentication enabled
```

Comment:
can we move the `# MySQL instance with IAM authentication enabled` comment above
```sh
export MYSQL_CONNECTION_NAME="<PROJECT>:<REGION>:<INSTANCE>"
```

Same with Postgres below...